contrib/mac-build

```diff
@@ -30,11 +31,15 @@ function reporterror {
 
 echo =============================================================================
 echo
-echo "running: sudo port install $macports_prereq"
 if which port &>/dev/null; then
+  echo "running: sudo port install $macports_prereq"
   echo "You may be asked for your password to install these packages."
   echo
   sudo port install $macports_prereq || reporterror
+elif which brew &>/dev/null; then
+  for package in $homebrew_prereq; do
+      brew list $package &>/dev/null || brew install $package || reporterror
```

Comment:
We should be really up front w/ installing prereqs for users.
1) What will they see? (Sorry for ignorance here.)
2) Should there be a [Y/n] prompt that can be forced off?